content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md

```diff
@@ -55,3 +55,11 @@ To obtain the Corda CLI installer:
    {{< note >}}
    If your PowerShell execution policy does not allow you to run this script, copy the contents to your own PowerShell script and execute that instead.
    {{< /note >}}
+
+## Verifying the Installation
+
+To verify your installation, run the following command:
+```shell
+corda-cli.sh -h
```

Comment:
`.sh` command will not be available on Windows, probably a good idea to split verification for *nix and Windows just like it was done above for the installation step.